pkg/corerp/frontend/controller/secretstores/kubernetes.go

```diff
@@ -195,6 +202,15 @@ func UpsertSecret(ctx context.Context, newResource, old *datamodel.SecretStore,
 		}
 	}
 
+	err = options.KubeClient.Create(ctx, &corev1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: ns}})
```

Comment:
Please use kubeutil for creating or updating namespace like below:

https://github.com/radius-project/radius/blob/275b861093057c9ded559b2ad36487647cab81a1/pkg/daprrp/processors/pubsubbrokers/processor.go#L93

Reply:
updated it